docs/content/Upgrade/manual/upgrade35.md

```diff
@@ -14,10 +14,10 @@ Using the PostgreSQL Operator {{< param operatorVersion >}} requires replacing y
 
 A major change to this container is that the PostgreSQL process is now managed by Patroni. This allows a PostgreSQL cluster that is deployed by the PostgreSQL Operator to manage its own uptime and availability, to elect a new leader in the event of a downtime scenario, and to automatically heal after a failover event.
 
-When creating your new clusters using version {{< param operatorVersion >}} of the PostgreSQL Operator, the `pgo create cluster` command will automatically use the new `crunchy-postgres-ha` image if the image is unspecified. If you are creating a PostGIS enabled cluster, please be sure to use the updated image name, as with the command:
+When creating your new clusters using version {{< param operatorVersion >}} of the PostgreSQL Operator, the `pgo create cluster` command will automatically use the new `crunchy-postgres-ha` image if the image is unspecified. If you are creating a PostGIS enabled cluster, please be sure to use the updated image name and image tag, as with the command:
 
 ```
-pgo create cluster mygiscluster --ccp-image=crunchy-postgres-gis-ha
+pgo create cluster mygiscluster --ccp-image=crunchy-postgres-gis-ha --ccp-image-tag=centos7-11.8-2.5-4.4.0
```

Comment:
Given this is for the 3.5 docs, do we need this here? This only affects 4.4+

Reply:
Yes, since the 3.5 manual upgrade calls for a cluster recreation, the specific GIS tag has to be provided as it would be for a brand new cluster.
I've updated the above to use the new params as well.